src/mechaphlowers/core/models/space_position_cable_models.py

```diff
@@ -115,3 +174,59 @@ def x(self, resolution: int = 10) -> np.ndarray:
 		end_points = self.x_n()
 
 		return np.linspace(start_points, end_points, resolution)
+
+	def L_m(self) -> np.ndarray:
+		p = self.p
+		x_m = self.x_m()
+		return -p * np.sinh(x_m / p)
+
+	def L_n(self) -> np.ndarray:
+		p = self.p
+		x_n = self.x_n()
+		return p * np.sinh(x_n / p)
+
+	def T_h(self) -> np.ndarray:
+		if self.linear_weight is None:
+			raise AttributeError("Cannot compute T_h: linear_weight is needed")
+		else:
+			p = self.p
+			m = self.load_coefficient
+			lambd = self.linear_weight
+			return p * m * lambd
```

Comment:
Since `lambd` is only used once, in a simple formula, maybe we don't need it? Or maybe we can call it just `linear_weight`?
(That's one way of solving the issue with "lambda" being a python reserved keyword)

```suggestion
			linear_weight = self.linear_weight
			return p * m * self.linear_weight
```